src/Interfaces/AbstractPolytope.jl

```diff
@@ -112,7 +112,7 @@ end
 # given a polytope P, apply the linear map P to each vertex of P
 # it is assumed that the interface function `vertices_list(P)` is available
 @inline function _linear_map_vrep(M::AbstractMatrix, P::AbstractPolytope)
-    vertices = broadcast(v -> M * v, vertices_list(P))
+    vertices = broadcast(v -> M * v, vertices_list(P)) |> unique
```

Comment:
i would rather optionally apply a convex hull on the function that calls this one, because:

- making it `false` by default, because in high dims we would need to run a possibly expensive convex hull
- it is more robust than `unique`
- the lower level `_linear_map_vrep` does one simple task (v -> Mv) keeping the simplification algorithm in a separate step

Reply:
Okay about `convex_hull` instead of `unique`, but:
* Given the other comment about `VPolygon` I would pass the option to `_linear_map_vrep` because in the `VPolygon` version we need to pass that option to the constructor.
* We do a `convex_hull` call in `vertices_list` for the lazy `LinearMap` by default as well. Is it okay to have this different behavior?

Reply:
> Given the other comment about VPolygon I would pass the option to _linear_map_vrep because in the VPolygon version we do not want to apply a convex hull again even if it is set to true.

ok, i see :+1: 

> We do a convex_hull call in vertices_list for the lazy LinearMap by default as well. Is it okay to have this different behavior?

i see, you mean `vertices_list(lm::LinearMap; prune::Bool=true)`. yeah i don't know, maybe that one should also be set to false..